src/Documentation~/compatibility.md

```diff
@@ -0,0 +1,15 @@
+# Compatibility with `Mathf`
+
+Unity Mathematics isn't implemented in the same way as the [`UnityEngine.Mathf`](https://docs.unity3d.com/ScriptReference/Mathf.html) struct in the core Unity engine. Because of this, if your application relies on the specific behaviors of `Mathf`, it might not be possible to use Unity Mathematics in your project. Also, if your project uses `Mathf` functions that have no corresponding Unity Mathematics function, or your code uses external plug-ins that depend on Unity types such as `Vector3`, or `Matrix4x4`, then you shouldn't use Unity Mathematics.
```

Comment:
We should clarify that you can still use Unity.Mathematics but you will have to reimplement those Mathf methods if you want similar behavior. Additionally, there is nothing that prevents users from calling into the Mathf methods and also using Unity.Mathematics at the same time (but you may need to pay a conversion cost for Vector3 to float3 (or vice versa).

I expect that for many users using Burst, the best way to migrate is to use Unity.Mathematics by default and only use Mathf (which will require paying the extra cost of converting between the UnityEngine math types and Unity.Mathematics types) when necessary.